inlong-agent/conf/agent.properties

```diff
@@ -108,18 +108,12 @@ agent.manager.vip.http.host=127.0.0.1
 agent.manager.vip.http.port=8083
 agent.manager.auth.secretId=
 agent.manager.auth.secretKey=
-
-
-
 ############################
-# prometheus config
+# metric config
 ############################
-# whether to enable prometheus
-agent.prometheus.enable=true
-# prometheus exporter server default port
+agent.domainListeners=org.apache.inlong.agent.metrics.AgentPrometheusMetricListener
```

Comment:
please list all available listeners in the annotation.

Reply:
resolve